bridge/bridge.js

```diff
@@ -0,0 +1,44 @@
+let MiniAppBridge = {}
+var queue = []
+var isPlatform = {
+    Android: function() {
+        return navigator.userAgent.match(/Android/i);
+    },
+    iOS: function() {
+        return navigator.userAgent.match(/iPhone|iPad|iPod/i);
+    }
+};
+
+MiniAppBridge.exec = function(action, onSuccess, onError) {
+    const callback = {}
+    callback.onSuccess = onSuccess;
+    callback.onError = onError;
+    callback.id = Math.random();
+    queue.unshift(callback)
+    if(isPlatform.iOS()){
+        webkit.messageHandlers.MiniApp.postMessage(JSON.stringify({action: action, id: callback.id}));
+    } else {
+        window.MiniApp.getUniqueId(JSON.stringify({action: action, id: callback.id}))
+    }
+}
+
+MiniAppBridge.execCallback = function(messageId, value) {
+    var queueObj = queue.find(callback => callback.id = messageId)
+    queueObj.onSuccess(value);
+    var messageObjIndex = queue.indexOf(queueObj)
+    if(messageObjIndex != -1) {
+        queue.splice(messageObjIndex, 1);
+    }
+}
+
+window.MiniApp = {
```

Comment:
Actually, we don't need `window.` here. Just `MiniApp` is enough for code running and it's also on the same page with TypeScript [interface](https://github.com/rakutentech/js-miniapp/blob/c0200d459403dfd2c38db87976ab91a342c307ba/src/miniapp.ts)

Reply:
Actually, I think we should be declaring `MiniApp` in the JS SDK, and then the calls in the JS SDK will basically be just wrapping calls to this one. So maybe this injected code should only declare a `MiniAppBridge` object?
```
window.MiniAppBridge = {};
MiniAppBridge.exec = ...
MiniAppBridge.execCallback = ...
MiniApp.getUniqueId = ....
```

And then in the JS SDK, the code will be something like:
```
window.MiniApp = {
   getUniqueId: () => MiniAppBridge.getUniqueId()
```

Does that make sense? What do you think?

Reply:
Changes are done.  Now the JS SDK can access like below,

```
window.MiniApp = {
   getUniqueId: () => MiniAppBridge.getUniqueId()
}
```